bin/steps/pip-install

```diff
@@ -58,7 +60,12 @@ if [ ! "$SKIP_PIP_INSTALL" ]; then
         exit 1
     fi
 
-    # Smart Requirements handling
+    # Cache pip packages ourselves
+    ## TODO: Update to cache pip's built-in cache
+
+    # # Update cached version of requirements.txt
+    # cp -R $BUILD_DIR/requirements.txt "$CACHE_DIR/.heroku/requirements.txt"
```

Comment:
My personal preference is to delete commented out code instead of leaving it. For the TODO, it might be better served in a tracker like the issues.